package-lock.json

```diff
@@ -1,271 +1,264 @@
 {
-  "name": "clic-website-v2",
-  "lockfileVersion": 3,
   "requires": true,
-  "packages": {
-    "": {
-      "devDependencies": {
-        "@commitlint/cli": "^17.4.4",
-        "@commitlint/config-conventional": "^17.4.4",
-        "@trivago/prettier-plugin-sort-imports": "^4.1.0",
-        "husky": "^8.0.0",
-        "prettier": "2.8.4"
-      }
-    },
-    "node_modules/@babel/code-frame": {
-      "version": "7.18.6",
-      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.18.6.tgz",
-      "integrity": "sha512-TDCmlK5eOvH+eH7cdAFlNXeVJqWIQ7gW9tY1GJIpUtFb6CmjVyq2VM3u71bOyR8CRihcCgMUYoDNyLXao3+70Q==",
-      "dev": true,
-      "dependencies": {
-        "@babel/highlight": "^7.18.6"
-      },
-      "engines": {
-        "node": ">=6.9.0"
+  "lockfileVersion": 1,
+  "dependencies": {
```

Comment:
Same comment regarding the npm version. With a set of changes that does not change the dependencies (i.e. package.json), the package-lock normally shouldn't change. Let's investigate together next time we meet 👍